app/domain/authentication/authenticator_input.rb

```diff
@@ -24,17 +25,17 @@ def update(hash)
 
     def webservice
       @webservice ||= ::Authentication::Webservice.new(
-        account:            @account,
-        authenticator_name: @authenticator_name,
-        service_id:         @service_id
+        account:            account,
+        authenticator_name: authenticator_name,
+        service_id:         service_id
```

Comment:
why did this change?

Reply:
API changes. The class variables in this current implementation are nill.

Reply:
which api changes? is this because you upgraded `dry`?